packages/components/time-picker/__tests__/time-picker.test.ts

```diff
@@ -243,6 +245,9 @@ describe('TimePicker', () => {
           onBlur(e) {
             return blurHandler(e)
           },
+          onKeydown(e) {
+            return keydownHandler(e)
+          },
         },
       }
     )
```

Comment:
Need to add some 'expect' statements to each of these tests, otherwise nothing is tested.

Something like this:

```
input.trigger('keydown')
await nextTick()
expect(keydownHandler).toHaveBeenCalledTimes(1)
```